src/Tizen.GenericTextClassifier/Tizen.GenericTextClassifier/GenericTextClassifier.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+
+namespace Tizen.GenericTextClassifier
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct CategoryListStruct
+    {
+        internal char[] first_category;
+        internal char[] second_category;
+        internal char[] third_category;
+        internal float first_category_score;
+        internal float second_category_score;
+        internal float third_category_score;
+    }
+
+    class GenericTextClassifier
+    {
+        internal const string LogTag = "Tizen.GenericTextClassifier";
+        /*int text_classifier_initialize(void);*/
+        public int TextClassifierInitialize()
+        {
+            int ret;
+
+            ret = Interop.GTCAPI.TextClassifierInitialize();
```

Comment:
It seem to wrong design, 
Because, `Interop.GTCAPI.TextClassifierInitialize()` is working as globally on process. 
But this API bounded on a instance object. 
So, if instance was created more than one, It does not work propertly